src/upgrades.cairo

```diff
@@ -66,10 +67,14 @@ mod Upgrades {
                     let mut airdrop_component_state: ComponentState<ContractState> =
                         Governance::airdrop_component::unsafe_new_component_state();
                     airdrop_component_state.merkle_root.write(impl_hash);
+                } else if (contract_type == 4) {
+                    let new_amm_address: ContractAddress = impl_hash.try_into().unwrap();
+                    let mut state = Governance::unsafe_new_contract_state();
+                    state.amm_address.write(new_amm_address);
                 } else {
                     assert(
-                        contract_type == 4, 'invalid contract_type'
-                    ); // type 4 is no-op, signal vote
+                        contract_type == 5, 'invalid contract_type'
+                    ); // type 5 is no-op, signal vote
```

Comment:
Why change the contract type of signal vote when we can just add 5 as the new prop type?